src/storage/mod.rs

```diff
@@ -696,6 +696,10 @@ impl<E: Engine, L: LockManager, F: KvFormat> Storage<E, L, F> {
                         wait_wall_time_ms: duration_to_ms(wait_wall_time),
                         process_wall_time_ms: duration_to_ms(process_wall_time),
                     };
+                    with_tls_tracker(|tracker| {
+                        tracker.metrics.read_pool_schedule_wait_nanos =
```

Comment:
This way looks okay now. In the future, I think we can unify the pool schedule wait durations by letting the read pool record this into the tracker.

Reply:
Yes, now it seems that the metric on the TiKV side is a little bit duplicated in some fields due to some compatibility issues.